redshift_sqlalchemy/dialect.py

```diff
@@ -484,7 +487,8 @@ def get_unique_constraints(self, connection, table_name,
         """
         Return information about unique constraints in `table_name`.
 
-        See :meth:`~sqlalchemy.engine.interfaces.Dialect.get_unique_constraints`.
+        See
+        :meth:`~sqlalchemy.engine.interfaces.Dialect.get_unique_constraints`.
```

Comment:
I don't think we can cross reference like this. I think a better approach is to link to the docs using HYPERTEXT